example/docker-compose.yml

```diff
@@ -24,41 +12,38 @@ services:
         condition: service_healthy
     restart: unless-stopped
     ports:
-      - ${WIKIBASE_PORT}:80
+      - ${WIKIBASE_PORT:-8880}:80
```

Comment:
closing this here to move the PR forward. follow up ticket is https://phabricator.wikimedia.org/T365038

Reply:
I am all for removing these, and will do it in this branch now. I totally agree and think this will be a lot more clear. A lot of these changes were happening in somewhat conservative steps.  Then things like this become revealed, and I'm like-- oh yeah--that can go away entirely now 👍🏼

Reply:
Done ✔️